packages/preview/echarm/0.1.0/typst.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "echarm"
+version = "0.1.0"
+entrypoint = "lib.typ"
+authors = ["lublak"]
+license = "MIT"
+description = "Run echarts in typst with the use of CtxJS."
+
```

Comment:
the examples directory should be excluded via `exclude = ["examples"]`

Reply:
Does it also exclude the image from the universe? I use them for the readme

Reply:
No, it's only for the compiler. Universe will load the image directly from GitHub.